File: Codigos estudiantes por lenguaje/Java/Daniel Cangrejo/Matematicas/MCD_&_MCM.java
import java.io.PrintWriter;
import java.util.*;


public class Main {
	//MAXIMO COMUN DIVISOR
	static int gcd(int a, int b) {
		return b==0?a:gcd(b, a%b);
	} 
	
	//MINIMO COMÚN MULTIPLO
	static int lcm(int a, int b) {
		return a*(b/gcd(a,b));
	}
	
	public static void main(String[] args) {
		int a = 80;
		int b = 30;
		System.out.println("GCD: " + gcd(a, b));
		System.out.println("LCM: " + lcm(a, b));
	
	}
}

File: Codigos estudiantes por lenguaje/C++/Daniel Cangrejo/Matematicas/MCD_&_MCM.cpp
#include<bits/stdc++.h>
#include<cstdlib>
using namespace std;
int gcd(int a,int b){
    return (b==0?a:gcd(b,a%b));
}
int lcm(int a,int b){
    return (a*(b/gcd(a,b)));
}
int main() {
    int a=60;
    int b=70;
    cout<<"GCD : "<<(gcd(a,b))<<endl;
    cout<<"LCM : "<<(lcm(a,b))<<endl;
    cout<<"GCD Interno:"<<(__gcd(a,b))<<endl;
    return 0;
}
